examples/angular/patches/@angular+bazel+9.0.0-next.8.patch

```diff
@@ -25,14 +39,23 @@ index 9d3c491..3c99746 100755
    fs.writeFileSync(output, JSON.stringify(data));
  }
 diff --git a/node_modules/@angular/bazel/src/ng_module.bzl b/node_modules/@angular/bazel/src/ng_module.bzl
-index 9b88fbb..68217d0 100755
+index 9b88fbb..54bf2c7 100755
 --- a/node_modules/@angular/bazel/src/ng_module.bzl
 +++ b/node_modules/@angular/bazel/src/ng_module.bzl
-@@ -20,6 +20,7 @@ load(
+@@ -13,13 +13,14 @@ load(
+     "DEFAULT_NG_COMPILER",
+     "DEFAULT_NG_XI18N",
+     "DEPS_ASPECTS",
+-    "NodeModuleSources",
++    "NpmPackageInfo",
+     "TsConfigInfo",
+-    "collect_node_modules_aspect",
++    "node_modules_aspect",
+     "compile_ts",
      "ts_providers_dict_to_struct",
      "tsc_wrapped_tsconfig",
  )
-+load("@build_bazel_rules_nodejs//:providers.bzl", "transitive_js_ecma_script_module_info")
++load("@build_bazel_rules_nodejs//:providers.bzl", "js_ecma_script_module_info", "js_named_module_info")
```

Comment:
do you mean to revert that last change?

Reply:
yes. I set it to `transitive_js_ecma_script_module_info` to verify the alias worked in the alias PR and now it can be switched back.